channelz/service/service.go

```diff
@@ -187,12 +190,13 @@ func securityToProto(se credentials.ChannelzSecurityValue) *channelzpb.Security
 		otherSecurity := &channelzpb.Security_OtherSecurity{
 			Name: v.Name,
 		}
-		if anyval, err := ptypes.MarshalAny(v.Value); err == nil {
+		if anyval, err := anypb.New(proto.MessageV2(v.Value)); err == nil {
```

Comment:
A recent [commit](https://github.com/protocolbuffers/protobuf-go/commit/e344383da66ac9cf11000f824674793a77fc7d7e) to the new protobuf library adds conversion method from v1 to v2 and vice-versa. This would really help us to reach our eventual goal of not having to import `github.com/golang/protobuf/proto`, and only import `google.golang.org/protobuf/proto`. 

We have another [outstanding PR](https://github.com/grpc/grpc-go/pull/6132) which updates our dependencies, and once this is merged, our dependency on `google.golang.org/protobuf` would switch to `v1.30.0`, which contains the above mentioned fix. Would you be OK to wait for this PR to be merged so that you can use `protoadapt.MessageV2Of` here instead of `proto.MessageV2`?

Reply:
For sure i can wait, it would be better to use `protoadapt.MessageV2Of`

Reply:
https://github.com/grpc/grpc-go/pull/6132 is merged now, and it updated gRPC-Go's dependency on `google.golang.org/protobuf` to `v1.30.0`. Now you should be able to use the protoadapt package here.

Reply:
Done